tasks/gaia.js

```diff
@@ -191,7 +177,7 @@ function startLocalNode(
   nodeOneId = ``
 ) {
   return new Promise((resolve, reject) => {
-    let command = `${nodeBinary} start --home ${nodeHome}` // TODO add --minimum_fees 1STAKE here
+    let command = `${nodeBinary} start --home ${nodeHome}` // TODO add --minimum_fees 1stake here
```

Comment:
Create tracking issue for this `TODO`

Reply:
https://github.com/cosmos/voyager/issues/1721